EvolvSwiftSDK/EvolvSwiftSDKTests/Common-Tests/EvolvStoreNewTests.swift

```diff
@@ -164,4 +195,43 @@ class EvolvStoreNewTests: XCTestCase {
         
         waitForExpectations(timeout: 2)
     }
+    
+    func testActiveKeysDoNotNotifyIfNoContextChangeHappened() {
+        let context = EvolvContextImpl(remoteContext: ["location" : "UA",
+                                                       "view" : "home",
+                                                       "signedin" : "yes"],
+                                       localContext: [:])
+        let evolvStore = initializeEvolvStore(with: context)
+        
+        let expectation = self.expectation(description: "Active keys will not sink twice.")
+        
+        var sinkedTimes = 0
+        evolvStore.activeKeys.sink { _ in
+            sinkedTimes += 1
+        }.store(in: &cancellables)
+        
+        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
+            if sinkedTimes == 1 { expectation.fulfill() }
+        }
+        
+        let isContextChanged = evolvStore.set(key: "signedin", value: "yes", local: false)
```

Comment:
Do we get the same behavior if I changed the context, but the active keys didn't change? e.g.
evolvStore.set(key: "anotherkey", value: "yes", local: false)